website/src/hugo/content/changelog.md

```diff
@@ -13,6 +13,7 @@ it.
 This is a maintenance release.  It is binary compatible with 0.23.4, and includes the changes in 0.22.5.
 
 Scala.js support is backported for a large subset of the modules present in 1.0.
+Additional Scala.js-only modules for using http4s in the browser have been spun off as https://github.com/http4s/http4s-dom.
```

Comment:
Will these be published against 0.23, too?

Reply:
Yes! See https://github.com/http4s/http4s-dom/issues/11.

Versioning starts to get a bit complicated, would appreciate any pointers. Also the jdk-http-client module is a good role model.